third_party/intel/lib/TritonGENToLLVM/TritonGENToLLVMPass.cpp

```diff
@@ -231,6 +231,12 @@ static bool isOCLBuiltinAvailable(TritonGEN::Matrix2DBlockLoadOp op) {
       op.getTileWidth() == 32 && op.getVBlocks() == 1)
     return false;
 
+  // Missing intel_sub_group_2d_block_read_8b_8r16x and
+  // intel_sub_group_2d_block_read_8b_16r16x
```

Comment:
If we plan to only request `8r16x1c` and `16r16x1c`, then
```suggestion
  // Missing intel_sub_group_2d_block_read_8b_8r16x1c and
  // intel_sub_group_2d_block_read_8b_16r16x1c
```